app/main.py

```diff
@@ -1,8 +1,35 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+
+        Person.people[name] = self
+
+    def set_relationships(self, wife: str = None, husband: str = None) -> None:
+        if wife and wife in Person.people:
+            self.wife = Person.people[wife]
+        if husband and husband in Person.people:
+            self.husband = Person.people[husband]
```

Comment:
wife and husband will be in people for sure because you firstly create all persons